wordpress-mobile/WordPress-iOS
# New Discover Welcome Banner

Part of #14399

This PRs adds the welcome banner the first time the user sees the Discover section.

### iPhone

| Light  | Dark |
| ------------- | ------------- |
| ![Simulator Screen Shot - iPhone 11 - 2020-07-14 at 09 49 04](https://user-images.githubusercontent.com/7040243/87429451-2d21cc00-c5ba-11ea-81cf-936ea30d927e.png) | ![Simulator Screen Shot - iPhone 11 - 2020-07-14 at 09 48 54](https://user-images.githubusercontent.com/7040243/87429498-3f9c0580-c5ba-11ea-9437-7933017d146c.png) |


### iPad

| Light  | Dark |
| ------------- | ------------- |
| ![Simulator Screen Shot - iPad Pro (9 7-inch) - 2020-07-14 at 09 53 01](https://user-images.githubusercontent.com/7040243/87429556-57738980-c5ba-11ea-9aab-3bd7f94b466d.png) | ![Simulator Screen Shot - iPad Pro (9 7-inch) - 2020-07-14 at 09 51 51](https://user-images.githubusercontent.com/7040243/87429581-60645b00-c5ba-11ea-8a3c-1dd2cd122629.png) |

### To test

**Make sure that you have overridden the Reader Improvements Phase 2 Feature Flag (Me -> App Settings -> Debug -> "Reader Improvements Phase 2" should be ON)**

1. Go to the Discover section
2. Check that the welcome banner appear at the top
3. Change to another section
4. Go back, check that the welcome banner doesn't appear

PR submission checklist:
